app/jobs/ingest_archival_media_bag_job.rb

```diff
@@ -66,22 +66,41 @@ def ingest
         component_groups.each do |cid, sides|
           media_resource = find_or_create_media_resource(cid)
           media_resource_change_set = MediaResourceChangeSet.new(media_resource, source_metadata_identifier: media_resource.source_metadata_identifier.first)
+          add_av(media_resource_change_set, sides)
+          add_pbcore(media_resource_change_set, sides)
+          media_resource_change_set.member_of_collection_ids += Array.wrap(collection.id)
+          changeset_persister.save(change_set: media_resource_change_set)
+        end
+      end
+
+      private
+
+        def add_av(media_resource_change_set, sides)
           sides.each do |side|
             file_set = create_av_file_set(side)
             media_resource_change_set.member_ids += Array.wrap(file_set.id)
             media_resource_change_set.sync
           end
-          media_resource_change_set.member_of_collection_ids += Array.wrap(collection.id)
-          changeset_persister.save(change_set: media_resource_change_set)
         end
-      end
 
-      private
+        def add_pbcore(media_resource_change_set, sides)
```

Comment:
I considered recommending adding the pbcore file_set to a different property, but this is how we do it with Geo so seems fine.

Reply:
we do get some useful behavior for free this way (listings, downloads)